plugins/org.jkiss.dbeaver.ext.cubrid/plugin.xml

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+
+<plugin>
+    
+    <extension point="org.jkiss.dbeaver.generic.meta">
+        <meta id="cubrid" class="org.jkiss.dbeaver.ext.cubrid.model.meta.CubridMetaModel" driverClass="cubrid.jdbc.driver.CUBRIDDriver"/>
+    </extension>
+    
+    <extension point="org.jkiss.dbeaver.mavenRepository">
+        <repository id="cubrid-maven-repo" name="Cubrid Repository" url="https://maven.cubrid.org">
+            <scope group="cubrid"/>
+        </repository>
+    </extension>
+    
+    <extension point="org.jkiss.dbeaver.dataSourceProvider">
+        <datasource
+            class="org.jkiss.dbeaver.ext.cubrid.CubridDataSourceProvider"
+            description="Cubrid JDBC connector"
+            id="cubrid"
+            label="Cubrid"
+            icon="#database_icon_default"

```

Comment:
I don't know where '#database_icon_default' icon is used, but shouldn't I use 'cubrid icon.png'?
Other plugins use icons for the same purpose.

Reply:
Yes, I think we should use 'cubrid_icon.png'. I didn't notice this during the development. Now, I have just updated it.